### PR TITLE
[CollectionHubs] various hubs qa

### PR DESCRIPTION
**Pull request by xtina-starr**
This address the following QA items:

- fixes border and spacing for hub modules
- adds mobile styles for FeaturedCollectionsRail

**Collection Hubs**
Screen Shot 2019-08-13 at 12 03 05 PM

**Mobile FeaturedCollectionsRail**
Screen Shot 2019-08-13 at 12 02 17 PM

### CODE REVIEW

File: src/Components/CollectionsHubRails/FeaturedCollectionsRails/index.tsx [hunk 1]
```diff
@@ -153,13 +153,17 @@ export const FeaturedCollectionsRailsContainer = createFragmentContainer(
...
 
-const Container = styled(Box)`
+const FeaturedCollectionsContainer = styled(Box)`
+  border-top: 1px solid ${color("black10")};
+`
+
+const Container = styled(Box)<{ width: string }>`
   border: 1px solid ${color("black10")};
-  width: 365px;
+  width: ${({ width }) => width};
```
Comment (damassi) [inline src/Components/CollectionsHubRails/FeaturedCollectionsRails/index.tsx]: `Box` should [already contain width](https://github.com/artsy/palette/blob/master/packages/palette/src/elements/Box/Box.tsx#L73)

File: src/Components/CollectionsHubRails/FeaturedCollectionsRails/index.tsx [hunk 2]
```diff
@@ -153,13 +153,17 @@ export const FeaturedCollectionsRailsContainer = createFragmentContainer(
...
 `
 
-const ImageContainer = styled(Flex)`
-  height: 280px;
+const ImageContainer = styled(Flex)<{ height: string }>`
+  height: ${({ height }) => height};
```
Comment (damassi) [inline src/Components/CollectionsHubRails/FeaturedCollectionsRails/index.tsx]: And Flex has height: https://github.com/artsy/palette/blob/master/packages/palette/src/elements/Flex/Flex.tsx#L20

File: src/Components/CollectionsHubRails/FeaturedCollectionsRails/index.tsx (hunk 1 above)
Comment (xtina-starr) [reply src/Components/CollectionsHubRails/FeaturedCollectionsRails/index.tsx]: Thanks for pointing this out!

**Comment by artsyit** [comment]
:rocket: PR was released in v16.24.3 :rocket:

**Pull request merged**